mantle/Makefile

```diff
@@ -11,9 +11,11 @@ schema-update:
 	$(MAKE) -C ../tools schema
 
 .PHONY: install
-install: bin/ore bin/kola bin/plume bin/kolet
+install: build
```

Comment:
See [`281c689` (#2372)](https://github.com/coreos/coreos-assembler/pull/2372/commits/281c689886b5cc720ccd17a2706f7993f340ef68) related to this.